File: core/src/assets/images/space/ships/ShipsResources.java
package assets.images.space.ships;

public class ShipsResources {

    public String spaceShipPNG(){
        return this.getClass().getResource("space-ship.png").getPath();
    }
    public String spaceShuttle_1PNG() {
        return this.getClass().getResource("space-shuttle_1.png").getPath();
    }
    public String spaceShuttle_2PNG() {
        return this.getClass().getResource("space-shuttle_2.png").getPath();
    }
}

File: core/src/assets/tempStuff/images/space/ships/ShipsResources.kt
package assets.tempStuff.images.space.ships

class ShipsResources {

    fun spaceShipPNG(): String {
        return this.javaClass.getResource("space-ship.png").path
    }

    fun spaceShuttle_1PNG(): String {
        return this.javaClass.getResource("space-shuttle_1.png").path
    }

    fun spaceShuttle_2PNG(): String {
        return this.javaClass.getResource("space-shuttle_2.png").path
    }
}